sdk/appcenter/src/test/java/com/microsoft/appcenter/utils/context/AuthTokenContextTest.java

```diff
@@ -118,8 +121,8 @@ public void setAuthTokenTest() {
         mAuthTokenContext.setAuthToken(null, null, null);
 
         /* Verify that listener is called on empty token. */
-        verify(mockListener, times(1)).onNewAuthToken(isNull(String.class));
-        verify(mockListener, times(1)).onNewUser(isNull(String.class));
+         verify(mockListener).onNewAuthToken(isNull(String.class));
```

Comment:
Code is not formatted.
I have to insist on using Android Studio commit dialog as instructions are not followed otherwise, this is not the first time.

Reply:
This is changed by @jwallra. I have fixed it.